### DISCUSSION TITLE
Apache NetBeans 20 release candidates

### DISCUSSION THREAD

**Discussion by neilcsmith-net** [apache/netbeans#6581] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

Changes from NetBeans 19 can be seen at https://github.com/apache/netbeans/releases/tag/20-rc1

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by lkishalmi** [comment]
It is also available as a Snap package on the edge channel:
https://snapcraft.io/netbeans
![image](https://github.com/apache/netbeans/assets/1381701/e8ea7277-816a-4953-a168-1b37b6c68fee)

**Comment by neilcsmith-net** [comment]
The second release candidate of Apache NetBeans 20 is now available from the links above.

Changes since 20-rc1 are listed at https://github.com/apache/netbeans/releases/tag/20-rc2

**Comment by neilcsmith-net** [comment]
The third release candidate of Apache NetBeans 20 is now available from the links above.

Changes since 20-rc2 are listed at https://github.com/apache/netbeans/releases/tag/20-rc3

**Comment by neilcsmith-net** [comment]
The fourth release candidate of Apache NetBeans 20 is now available from the links above.

Changes since 20-rc2 are listed at https://github.com/apache/netbeans/releases/tag/20-rc4

I've added a signed macOS installer package to the nightlies installer links for testing.

**Comment by mbien** [comment]
unpinning this thread, NetBeans 20 is released and available from the usual place:
https://netbeans.apache.org/download/
